build-conf/REXX.properties

```diff
@@ -0,0 +1,56 @@
+# Releng properties used by language/REXX.groovy
+
+#
+# Comma separated list of required build properties for REXX.groovy
+REXX_requiredBuildProperties=REXX_srcPDS,REXX_objPDS,REXX_loadPDS,\
+  REXX_compiler,REXX_linkEditor,REXX_tempOptions,applicationOutputsCollectionName,\
```

Comment:
applicationOutputsCollectionName seems not to be used by the script

Reply:
I removed that one